builders/ethereum/dev-env/ape.md

```diff
@@ -343,4 +343,18 @@ Once completed, you should see a transaction hash and a value of `4` printed to
 
 Congratulations! You have successfully deployed and interacted with a contract using Ape!
 
+## Using Ape with a Local Node {: #using-ape-with-a-local-node }
```

Comment:
Line 340: this text says I should see a transaction hash and a value of `4` printed to the console but the terminal snippet says Stored value: 5

Reply:
thx good catch